<launch> 
	<!-- Stitches together point clouds of a single scan into a point cloud of a single one-way sweep of the front wobbler, based on motor angle. -->
	<node pkg= "babs_lidar_wobbler" type= "hokuyo_pcl_stitcher" name= "front_hokuyo_pcl_stitcher" output="log">

		<remap from="/front_hokuyo_pcl_stitcher/scan_cloud" to="/front_wobbler/scan_cloud"/>
		<remap from="/front_hokuyo_pcl_stitcher/point_cloud" to="/front_wobbler/point_cloud"/>
		<remap from="/front_hokuyo_pcl_stitcher/angle" to="/front_wobbler/cmd"/>
	</node>

	<!-- Stitches together point clouds of a single scan into a point cloud of a single one-way sweep of the rear wobbler, based on motor angle. -->
	<node pkg= "babs_lidar_wobbler" type= "hokuyo_pcl_stitcher" name= "rear_hokuyo_pcl_stitcher" output="log">

		<remap from="/rear_hokuyo_pcl_stitcher/scan_cloud" to="/rear_wobbler/scan_cloud"/>
		<remap from="/rear_hokuyo_pcl_stitcher/point_cloud" to="/rear_wobbler/point_cloud"/>
		<remap from="/rear_hokuyo_pcl_stitcher/angle" to="/rear_wobbler/cmd"/>
	</node>
</launch>
